src/app.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express')
 
-const app = express()
+const routes = require('./routes');
 
-app.use(express.json())
+const app = express();
 
-app.get('/', (req, res) => res.send({ message: 'ok' }))
+app.use(express.json());
+app.use(routes);
 
-module.exports = app
+app.get('/', (req, res) => res.send({ message: 'ok' }));
```

Comment:
I think it's valid, send this app.get to routes.

Reply:
Sure is. Will do it tonight.

Reply:
Nice, GJ!